docs/quick-start-guide.md

```diff
@@ -50,6 +50,41 @@ $ kubectl describe deployment sparkoperator -n sparkoperator
 
 ```
 
+### Metrics
+
+The operator exposes a set of metrics via the metric endpoint to be scraped by `Prometheus`. This is enabled by default and can be disabled by setting  the flag `-enable-metrics=false` and running 
```

Comment:
I'm thinking maybe the this should default to `false` since not everyone is running Prometheus.

Reply:
done